core/src/main/java/org/infinispan/expiration/impl/TouchCommand.java

```diff
@@ -0,0 +1,115 @@
+package org.infinispan.expiration.impl;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.concurrent.CompletableFuture;
+
+import org.infinispan.commands.InitializableCommand;
+import org.infinispan.commands.TopologyAffectedCommand;
+import org.infinispan.commands.remote.BaseRpcCommand;
+import org.infinispan.commons.io.UnsignedNumeric;
+import org.infinispan.commons.time.TimeService;
+import org.infinispan.container.impl.InternalDataContainer;
+import org.infinispan.distribution.DistributionInfo;
+import org.infinispan.distribution.DistributionManager;
+import org.infinispan.distribution.LocalizedCacheTopology;
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.statetransfer.OutdatedTopologyException;
+import org.infinispan.util.ByteString;
+import org.infinispan.util.concurrent.CompletableFutures;
+
+/**
+ * This command can be invoked to update a cache entry's recent access. This can involve updating its last access
```

Comment:
`can` suggests to me that it is parametrized and can do each independently. I'd rather say 

`This command updates a cache entry's last access timestamp. If eviction is enabled, it will also update the recency information`.